-------------------------------------------------------------------------------
Test set: dao.Sql2oDepartmentsDaoTest
-------------------------------------------------------------------------------
Tests run: 7, Failures: 0, Errors: 7, Skipped: 0, Time elapsed: 16.25 sec <<< FAILURE! - in dao.Sql2oDepartmentsDaoTest
addUserToDepartment(dao.Sql2oDepartmentsDaoTest)  Time elapsed: 3.462 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)

getAll(dao.Sql2oDepartmentsDaoTest)  Time elapsed: 2.179 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)

getDepartmentNews(dao.Sql2oDepartmentsDaoTest)  Time elapsed: 2.524 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)

idSetForAddedDepartment(dao.Sql2oDepartmentsDaoTest)  Time elapsed: 3.07 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)

getAllUsersInDepartment(dao.Sql2oDepartmentsDaoTest)  Time elapsed: 2.354 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)

correctDepartmentIsReturnedFindById(dao.Sql2oDepartmentsDaoTest)  Time elapsed: 2.66 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.212.116.98", user "fvvikmppgjhovk", database "da93g9c21mukon", SSL off
	at dao.Sql2oDepartmentsDaoTest.setUp(Sql2oDepartmentsDaoTest.java:37)

dao.Sql2oDepartmentsDaoTest  Time elapsed: 2.66 sec  <<< ERROR!
java.lang.NullPointerException
	at dao.Sql2oDepartmentsDaoTest.shutDown(Sql2oDepartmentsDaoTest.java:50)

